bin/faucet/src/api/frontend.rs

```diff
@@ -0,0 +1,31 @@
+use axum::response::{Html, IntoResponse, Response};
+use axum_extra::response::{Css, JavaScript};
+use http::header::{self};
+
+pub async fn get_index_html() -> Html<&'static str> {
+    Html(include_str!("../../frontend/index.html"))
+}
```

Comment:
Could we make the path here relevant to something like `CARGO_MANIFEST_DIR`? For example:
```rust
Html(include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/frontend/index.html")))
```

Also, would be good to confirm that this will work fine during publishing (I think it should work, but wanted to double-check).